src/form/__tests__/analytics.test.tsx

```diff
@@ -83,7 +84,33 @@ describe('Form Analytics', () => {
     );
   });
 
-  test('does not send a funnelComplete when the form is unmounted after clicking a primary button in the children slot', () => {
+  test('does not send a funnelComplete metricv when the form is unmounted after clicking a primary button in a modal', () => {
```

Comment:
Typo - "metric" instead of "metricv":

```suggestion
  test('does not send a funnelComplete metric when the form is unmounted after clicking a primary button in a modal', () => {
```